packages/main/src/MenuItemGroup.ts

```diff
@@ -0,0 +1,127 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import MenuItem from "./MenuItem.js";
+import MenuItemGroupTemplate from "./generated/templates/MenuItemGroupTemplate.lit.js";
+import ItemSelectionMode from "./types/ItemSelectionMode.js";
+import type { IMenuItem } from "./Menu.js";
+
+/**
+ * @class
+ *
+ * ### Overview
+ *
+ * `ui5-menu-item-group` is the group of items to use inside a `ui5-menu`.
+ * Items that belong to the same group should be enclosed by `ui5-menu-item-group`.
+ * Each group can have itemSelectionMode property, which defines the selection mode of the items inside.
+ * Possible values are 'None' (default), 'SingleSelect', 'MultiSelect'.
+ *
+ * ### Usage
+ *
+ * `ui5-menu-item-group` represents a collection of `ui5-menu-item` components that can have the same selection mode.
+ * The items are addeed to the group's `items` slot.
+ *
+ * ### ES6 Module Import
+ *
+ * `import "@ui5/webcomponents/dist/MenuItemGroup.js";`
+ * @constructor
+ * @extends UI5Element
+ * @implements {IMenuItem}
+ * @since 2.4.0
+ * @public
+ */
+@customElement({
+	tag: "ui5-menu-item-group",
+	renderer: litRender,
+	template: MenuItemGroupTemplate,
+	dependencies: [MenuItem],
+})
+class MenuItemGroup extends UI5Element implements IMenuItem {
+	/**
+	 * Defines the component selection mode.
+	 * @default "None"
+	 * @public
+	 */
+	@property()
+	itemSelectionMode: `${ItemSelectionMode}` = ItemSelectionMode.None;
```

Comment:
same as in the MenuItem.

Add also since tag and use plain string as initializer.

Reply:
Same here, `selectionMode` belongs to **ListItem** with different meaning and behavior
Regarding the `@since` - the whole component is new, and already have `@since` tag.